app/src/main/java/com/hemdan/mvvm/ui/main/actorslist/ActorsListFragment.kt

```diff
@@ -0,0 +1,62 @@
+package com.hemdan.mvvm.ui.main.actorslist
+
+import android.os.Bundle
+import android.view.View
+import androidx.lifecycle.Observer
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.hemdan.mvvm.R
+import com.hemdan.mvvm.data.model.PopularInfo
+import com.hemdan.mvvm.ui.base.BaseFragment
+import kotlinx.android.synthetic.main.fragment_actors_list.*
+import java.util.ArrayList
+import javax.inject.Inject
+
+class ActorsListFragment : BaseFragment() {
+
+    private lateinit var listAdapter: ActorsAdapter
+    private var listLayoutManager: LinearLayoutManager? = null
```

Comment:
@MonaAliMahmoud you don't have to use this as class property or even nullable as it's not used in different code blocks/methods